src/io.h

```diff
@@ -27,12 +27,14 @@ template <class ThisPtcl> void OutputFileWithTimeInterval(PS::ParticleSystem<Thi
 		FileHeader header;
 		header.time = sysinfo.time;
 		header.Nbody = sph_system.getNumberOfParticleLocal();
-		char filename[256];
-        sprintf(filename, "results/%s/%05d", out_dir, sysinfo.output_id);
-		sph_system.writeParticleAscii(filename, "%s_%05d_%05d.dat", header);
+		char filename[20];
+        sprintf(filename, "results.%05d", sysinfo.output_id);
```

Comment:
I think this line will cause the files to be called something like `results.00000_00001_00000.dat`. Is this the intention?